lemmy/.env-dist

```diff
@@ -0,0 +1,29 @@
+# The domain name for the whoami service:
+LEMMY_TRAEFIK_HOST=lemmy.example.com
+LEMMY_VERSION=0.18.3
+LEMMY_UI_VERSION=0.18.3
+ASONIX_PICTRS_VERSION=0.4.1
```

Comment:
This var should start with LEMMY_

Reply:
I thought about that. I chose not to because it's pulling a non-Lemmy service even though it's in d.rymcg.tech/lemmy. But now that I think about it more, we do that all the time - so I'll change it.

Reply:
Yea the prefix is indicating which d.ry project the var is for, lemmy.

Reply:
I've never tested it, but d.ry is designed so that all the vars could go in one giant .env file, and there would be no naming conflicts because they are prefixed

Reply:
fixed in [21d70f3](https://github.com/EnigmaCurry/d.rymcg.tech/pull/92/commits/21d70f3d793f066ab97edbf562415b039fefc6dc)